modules/kubernetes/prometheus/charts/prometheus-extras/templates/prometheus.yaml

```diff
@@ -1,11 +1,13 @@
 apiVersion: monitoring.coreos.com/v1
 kind: Prometheus
 metadata:
+  name: prometheus-xks
```

Comment:
Should we change the name to `xks` instead? 
Right now the generated pods are called: `prometheus-prometheus-xks-0`